formats/v1.9/readme

```diff
@@ -24,4 +24,23 @@ We're putting these into version control in order to track our changes, which
 we hope will be relatively small. (Except for some prettyprinting we'll do,
 through an editor, as the first change.) Substantive changes are geared towards
 the goal of helping our automatic code-generation tool to work more smoothly.
-Note that summary_gpdc.json is being renamed to summary_containers.json.
+
+The following changes have been made:
```

Comment:
Kudos for adding to my old readme. :-) That shows some dedication.

Reply:
It'll be easier to remember what we changed :-)